docker.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+if [ -t 1 ]; then
+    tty_flag="-t"
+else
+    tty_flag=""
+fi
+dir=$(python -c 'import os; import sys; print os.path.abspath(sys.argv[1])' $(dirname "$0"))
```

Comment:
I do too, but that's Linux-only... I wanted the wrapper script to run on Mac as well so I changed it...